packages/constants/package.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "@tupaia/constants",
+  "version": "1.0.0",
+  "private": true,
+  "description": "Tupaia constants repository",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/beyondessential/tupaia.git",
+    "directory": "packages/constants"
+  },
+  "author": "Beyond Essential Systems <admin@tupaia.org> (https://bes.au)",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "installConfig": {
+    "hoistingLimits": "workspaces"
+  },
+  "scripts": {
+    "build": "rimraf dist && npm run --prefix ../../ package:build:ts",
+    "build-dev": "yarn run build",
+    "build-watch": "rimraf dist && npm run --prefix ../../ package:build:ts-watch",
```

Occurred to me this won’t be caught by Git conflict resolution:

```suggestion
    "build": "rimraf dist && yarn run package:build:ts",
    "build-dev": "yarn run build",
    "build-watch": "rimraf dist && yarn run package:build:ts-watch",
```